Rakefile

```diff
@@ -70,6 +68,26 @@ namespace :triannon do
   desc 'run test rails console w triannon but no jetty'
   task :console => :console_no_jetty
 
+  desc 'update triannon gems and engine cart app'
+  task :update_app do
```

Comment:
While I'm not sure this task is even necessary, I think you could simplify it to:

```
task :update_app => ['engine_cart:clean'] do
  system 'bundle update'
  # Rake::Task['engine_cart:generate'].invoke # if this is even needed; it'll be generated on-the-fly when needed anyway..
end
```